### PR TITLE
docs: documentation for at, equal, fold, filter, and group-by

### PR DESCRIPTION
**Pull request by pevisscher**

### CODE REVIEW

**Comment by pevisscher** [comment]
Current dependencies on/for this PR:
* main
  * **PR #30** <a href="https://app.graphite.dev/github/pr/skyleague/axioms/30" target="_blank">Graphite</a> 
    * **PR #31** <a href="https://app.graphite.dev/github/pr/skyleague/axioms/31" target="_blank">Graphite</a> 
      * **PR #32** <a href="https://app.graphite.dev/github/pr/skyleague/axioms/32" target="_blank">Graphite</a> 
        * **PR #33** <a href="https://app.graphite.dev/github/pr/skyleague/axioms/33" target="_blank">Graphite</a> 
          * **PR #34** <a href="https://app.graphite.dev/github/pr/skyleague/axioms/34" target="_blank">Graphite</a> 
            * **PR #35** <a href="https://app.graphite.dev/github/pr/skyleague/axioms/35" target="_blank">Graphite</a> 
              * **PR #36** <a href="https://app.graphite.dev/github/pr/skyleague/axioms/36" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/axioms/36?utm_source=stack-comment).

File: src/iterator/at/at.ts [hunk 1]
```diff
@@ -19,18 +40,93 @@ export function at<T, N extends number = number>(xs: Traversable<T>, n: N): Mayb
     return head(drop(xs, n))
 }
 
+/**
+ * Return the first element of a traversable, or nothing if the traversable is empty.
```
Comment (MickVanDuijn) [inline src/iterator/at/at.ts:44]:
```suggestion
 * Return the first element of a traversable, or Nothing if the traversable is empty.
```

File: src/iterator/at/at.ts [hunk 2]
```diff
@@ -19,18 +40,93 @@ export function at<T, N extends number = number>(xs: Traversable<T>, n: N): Mayb
...
+ * @group Iterators
+ */
 export function first<T>(xs: T): T extends readonly [infer N0, ...unknown[]] ? N0 : Nothing
 export function first<T>(xs: Traversable<T>): Maybe<T>
 export function first<T>(xs: Traversable<T>): Maybe<T> {
     return at(xs, 0)
 }
 
+/**
+ * Return the second element of a traversable, or nothing if the traversable is empty.
```
Comment (MickVanDuijn) [inline src/iterator/at/at.ts:75]:
```suggestion
 * Return the second element of a traversable, or Nothing if the traversable is empty.
```

File: src/iterator/at/at.ts [hunk 3]
```diff
@@ -19,18 +40,93 @@ export function at<T, N extends number = number>(xs: Traversable<T>, n: N): Mayb
...
+ * @group Iterators
+ */
 export function second<T>(xs: readonly [unknown, T, ...unknown[]]): T
 export function second<T>(xs: Traversable<T>): Maybe<T>
 export function second<T>(xs: Traversable<T>): Maybe<T> {
     return at(xs, 1)
 }
 
+/**
+ * Return the third element of a traversable, or nothing if the traversable is empty.
```
Comment (MickVanDuijn) [inline src/iterator/at/at.ts:106]:
```suggestion
 * Return the third element of a traversable, or Nothing if the traversable is empty.
```

**Review by MickVanDuijn — APPROVED** [review]

**Comment by pevisscher** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/axioms/36?utm_source=stack-comment).


Job Reference: K5yEkBD2dLgv1WFxOfGc

**Pull request merged**